test/integration/perfutil/util.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2015 The Kubernetes Authors.
```

Comment:
Please change this to 2018.

Reply:
makes sense, was retaining the old header as the code was a refactor from existing one. Will change it.